pandas/core/reshape/reshape.py

```diff
@@ -1072,6 +1072,234 @@ def get_empty_frame(data) -> DataFrame:
         return DataFrame(dummy_mat, index=index, columns=dummy_cols)
 
 
+def from_dummies(
+    data: DataFrame,
+    to_series: bool = False,
+    prefix_sep: str | list[str] | dict[str, str] = "_",
+    columns: None | Index | list[str] = None,
```

Comment:
columns -> subset, should be 2nd arg.
prefix_sep -> sep (or call it prefix)
to_series, kill this
dropped_first -> not sure about this name

Reply:
I tried to follow the argnames of the `get_dummies` function to highlight the parallels:
```python
def get_dummies(
    data,
    prefix=None,
    prefix_sep="_",
    dummy_na: bool = False,
    columns=None,
    sparse: bool = False,
    drop_first: bool = False,
    dtype: Dtype | None = None,
) -> DataFrame:
```
But looking at the `from_dummies` function on its own your suggestions make more sense. I will try to come up with a more self-explanatory name for `dropped_first`